tasks/source.ps1

```diff
@@ -0,0 +1,54 @@
+[CmdletBinding()]
+Param(
+  [string]$command = '',
+  [string]$name = '',
+  [string]$location = '',
+  [string]$user = '',
+  [string]$password = '',
+  [int]$priority = 0,
+  [boolean]$bypass_proxy = $false,
+  [boolean]$allow_self_service = $false,
+  [boolean]$admin_only = $false,
+  [string]$_task
+)
+
+if ($command -eq '') {
+  # $_task will be something like 'chocolatey::source_xxx'
```

Comment:
I suppose this is an idiomatic implementation for bolt tasks (to have `noun_verb` variations instead of just `noun` with a command parameter, but this should probably be callable directly and not via another task, I think?

ie this should be valid

```bash
bolt task run chocolatey::source command=add name=foo location=\\some\share
```

Reply:
This will, I think, make testing a bit easier as you can just verify that the wrapper tasks are sending the correct info to this one.